src/Button/BaseButton.js

```diff
@@ -2,8 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { buttonStyle, variant, style } from 'styled-system';
-import { COMMON, LAYOUT, POSITION, BORDERS, TYPOGRAPHY } from '../constants';
-import theme from '../theme';
+import { StyledBox } from '../Box';
```

Comment:
Why not import default `Box` here?

Reply:
Using `styled(Box)` and `styled(StyledBox)` does not have the same output. I can show you.